demo/src/app/pages/demo/components/advanced-example-types.component.ts

```diff
@@ -14,77 +15,88 @@ export class AdvancedExamplesTypesComponent {
       name: "Leanne Graham",
       username: "Bret",
       email: "Sincere@april.biz",
-      comments: "Lorem ipsum dolor sit amet, ex dolorem officiis convenire usu."
+      comments: "Lorem ipsum dolor sit amet, ex dolorem officiis convenire usu.",
+      custom: '10|20'
```

Comment:
yeah, good idea! There already is valuePrepareFunction and also something called rendereFunction, but those are just functions (which is not really an angular way), so probably we need to consider custom renderer component or something like that? But I believe this would be a separate feature.